File: tests/dynamic_fixtures/rust/xss_positive.rs
/// XSS — positive fixture.
///
/// Vulnerable function: echoes user input directly into HTML without escaping.
/// Expected verdict: Confirmed (XSS payload echoed verbatim to output).
/// Cap: HTML_ESCAPE  Entry: `run(payload: &str)`
pub fn run(payload: &str) {
    // Vulnerable: direct string interpolation into HTML output.
    println!("__NYX_SINK_HIT__");
    let _ = std::io::Write::flush(&mut std::io::stdout());
    let html = format!("<div class='comment'>{}</div>", payload);
    println!("{}", html);
}

File: tests/dynamic_fixtures/python/xss_positive.py
"""XSS — positive fixture.

Vulnerable function: echoes user input directly into HTML without escaping.
Expected verdict: Confirmed (XSS payload echoed verbatim to output).
"""


def render_comment(user_input):
    """Vulnerable: no HTML escaping."""
    html = f"<div class='comment'>{user_input}</div>"
    print(html)
